lib/workspaceSocket.js

```diff
@@ -27,24 +23,28 @@ module.exports.connection = function (socket) {
         console.log('Initializing remote workspace host.');
         workspace.controlContainer(workspace_id, 'init', (err) => {
             if (ERR(err, e => logger.error(e))) return;
-            module.exports._namespace.to(workspace_id).emit('host-ready');
+            
+            const state = 'running';
+            workspace.updateState(workspace_id, state, () => {
+                module.exports._namespace.to(workspace_id).emit('change:state', {workspace_id, state});
```

Comment:
this manual callback is only here right now because the `lib/workspace#updateState` sockets are currently broken